OracleManagementAgent/dockerfiles/latest/container-scripts/watchdog.sh

```diff
@@ -194,6 +194,54 @@ function is_agent_alive()
   return 1
 }
 
+###########################################################
+# Upsert properties with its overriding configMap
+function agent_prop_upsert()
+{
+  log "invoking agent property upsert"
+  # while upgrading from previous binary, the script will not be available
+  # before the upgrade completes
+  if [ ! -f $MGMTAGENT_HOME/agent_inst/bin/agent_prop_upsert.sh ]; then
+    log "skipping property upsert"
+  else
+    # if the agent is alive then stop it first and then do the property update
+    if is_agent_alive; then
+	  stop_agent
+    fi
+
+    # upsert emd.properties
+    prop_file=$MGMTAGENT_HOME/agent_inst/config/emd.properties
+    config_map=$BASE_DIR/mgmtagent_agent_config/emd.properties
+    eval "/bin/sh $MGMTAGENT_HOME/agent_inst/bin/agent_prop_upsert.sh $prop_file $config_map 'Modifiable Properties'"
+  fi
+}
+
+###########################################################
+# cleans up mgmt_agent dir if a new cleanup id is provided
+function cleanup_agent_dir()
+{
+	log "initiating mgmt_agent dir cleanup"
+	
+	if [ -v POD_CLEANUP_ID ]; then
+		# create the cleanup dir to maintain the marker files
+		cleanup_id_dir="$BASE_DIR/cleanup_ids"
+		mkdir -p $cleanup_id_dir
+		
+	    cleanup_id_file="$cleanup_id_dir/$POD_CLEANUP_ID.txt"
+	    
+	    if [ ! -f $cleanup_id_file ]; then
```

Comment:
There are a couple of lint issues found.  You need to quote the file paths.  Can you address those?

Reply:
done